### PR TITLE
Status cleanup

### PR DESCRIPTION
**Pull request by mootz12**

### CODE REVIEW

File: contracts/tests/tests/governor/test_propose.rs [hunk 1]
```diff
@@ -185,13 +224,127 @@ fn test_propose_snapshot() {
...
+    let bytes = BytesN::<32>::random(&e);
+    let action = ProposalAction::Upgrade(bytes);
+
+    let proposal_id = governor_client.propose(&samwise, &title, &description, &action);
+
+    let proposal = governor_client.get_proposal(&proposal_id).unwrap();
+    assert_eq!(proposal.id, 0);
+    assert_eq!(proposal.config.title, title);
+    assert_eq!(proposal.config.description, description);
+    matches!(proposal.config.action, ProposalAction::Upgrade(_));
```
Comment (Ryang-21) [inline contracts/tests/tests/governor/test_propose.rs:265]: Couldn't you just `assert_eq!` against `action`?

Comment (Ryang-21) [reply contracts/tests/tests/governor/test_propose.rs:265]: Same for the test below

**Review by Ryang-21 — CHANGES_REQUESTED** [review]
Left a couple comments on the propose tests

File: contracts/tests/tests/governor/test_propose.rs (hunk 1 above)
Comment (mootz12) [reply contracts/tests/tests/governor/test_propose.rs:265]: No, Rust `enums` don't have `PartialEq` derived automatically. Match is able to determine if it is the same enum type, tho.

We have tests the ensures the contents are applied correctly during execution, this is just a sanity check to ensure the proper proposal was proposed.

**Review by Ryang-21 — APPROVED** [review]
lgtm

**Pull request merged**